torch/fx/experimental/sym_node.py

```diff
@@ -240,6 +240,13 @@ def __str__(self):
         return self.str()
 
     def __repr__(self):
+        return (
+            f"SymNode(expr={self.expr}, shape_env={self.shape_env}, "
```

Comment:
Let's change this to only print `hint`, and `constant` if they're not None.
I would also omit the `expr=` on the first parameter (since it's required and important).